cloud-sql/postgres/knex/app.standard.yaml

```diff
@@ -17,6 +17,7 @@ runtime: nodejs18
 # The following env variables may contain sensitive information that grants
 # anyone access to your database. Do not add this file to your source control.
 env_variables:
+  INSTANCE_CONNECTION_NAME: <MY-PROJECT>:<INSTANCE-REGION>:<INSTANCE-NAME>
```

Comment:
same here I think we need to leave as `INSTANCE_UNIX_SOCKET` to not mess up quickstarts